apps/src/containedLevels.js

```diff
@@ -141,7 +142,7 @@ export function initializeContainedLevel() {
       stepButton.prop('disabled', !validResult);
       if (validResult) {
         runButton.qtip('hide');
```

Comment:
if qtip was already used why do we need to install it again?

Reply:
It was needed for the unit tests.